docs/source/algorithms/PDFFourierTransform-v2.rst

```diff
@@ -0,0 +1,189 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The algorithm transforms a single spectrum workspace containing 
+spectral density :math:`S(Q)`, :math:`S(Q)-1`, or :math:`Q[S(Q)-1]` 
+(as a function of **MomentumTransfer** or **dSpacing** :ref:`units <Unit Factory>`) to a PDF 
+(pair distribution function) as described below and also the reverse. The available PDF types are the
+reduced pair distribution function :math:`G(r)`, the pair distribution function :math:`g(r)`, and the
+radial distribution function :math:`RDF(r)`.
+
+The output from this algorithm will have an x-range between 0.0 and the maximum parameter of the output,
+i.e. if converting from `g(r)` to `S(Q)` the output will be between 0.0 and `Qmax`.
+
+The spectrum density should be in the Q-space (\ **MomentumTransfer**\ ) :ref:`units <Unit Factory>` .
+(d-spacing is not supported any more. Contact development team to fix that and enable **dSpacing** again)
+
+References
+----------
+
+#. B. H. Toby and T. Egami, *Accuracy of Pair Distribution Functions Analysis Appliced to Crystalline and Non-Crystalline Materials*, Acta Cryst. (1992) A **48**, 336-346
+   `doi: 10.1107/S0108767391011327 <http://dx.doi.org/10.1107/S0108767391011327>`_
+#. B.H. Toby and S. Billinge, *Determination of Standard uncertainties in fits to pair distribution functions*  Acta Cryst. (2004) A **60**, 315-317]
+   `doi: 10.1107/S0108767304011754 <http://dx.doi.org/10.1107/S0108767304011754>`_
+
+.. The algorithm itself is able to identify the unit.  -- not any more. TODO:  should be investigated why this has been disabled
+
+
+PDF Options
+--------------
+
+**G(r)**
```

Comment:
This comment got lost I think so just readding now...
In an ideal world the documentation here would start with g(r) and have the integral explained under that heading. Then lower down we would have sections for G(r) and RDF(r) explaining how they are calculated from g(r). If it's possible to squeeze that in today without a lot of effort please can you try and rearrange? If you run out of time we can pick up another time.
I think if you're explaining the integral under a g(r) heading then you would need a different factor on the front (the r dependence would be 1/r^3 instead of 1/r^2)